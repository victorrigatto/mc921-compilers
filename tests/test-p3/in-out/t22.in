int main() {
    int i=0;
    i=i+1;
    {
      char i;
      if(i > 1)
        return 0;
      else
        return 1;
    }
}
