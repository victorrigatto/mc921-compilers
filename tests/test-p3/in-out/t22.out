SemanticError: Binary operator > does not have matching LHS/RHS types @ 6:10
